internal/config/config.go

```diff
@@ -16,11 +17,12 @@ const (
 )
 
 type Config struct {
-	Database   *dbConfig         `json:"database,omitempty"`
-	Service    *svcConfig        `json:"service,omitempty"`
-	KV         *kvConfig         `json:"kv,omitempty"`
-	Auth       *authConfig       `json:"auth,omitempty"`
-	Prometheus *prometheusConfig `json:"prometheus,omitempty"`
+	Database   *dbConfig               `json:"database,omitempty"`
+	Service    *svcConfig              `json:"service,omitempty"`
+	KV         *kvConfig               `json:"kv,omitempty"`
+	Auth       *authConfig             `json:"auth,omitempty"`
+	Prometheus *prometheusConfig       `json:"prometheus,omitempty"`
+	CAConfig   *ca_config.CAConfigType `json:"caconfig,omitempty"`
```

Comment:
```suggestion
	CAConfig   *ca_config.CAConfigType `json:"ca,omitempty"`
```
I'd leave out "config", because everything in this file is config. Also, consider spelling this out (though we haven't done so with `kv`, either, unfortunately).

Reply:
Ack